### PR TITLE
fix: docker-compose now persists kafka broker data

### PR DESCRIPTION
**Pull request by icy**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ services:
     ports:
       - "29092:29092"
       - "9092:9092"
+    volumes:
+      - broker://var/lib/kafka/data/
```
Comment (dexter-mh-lee) [inline docker/docker-compose.yml:30]: can we remove the double slash?

Comment (icy) [reply docker/docker-compose.yml]: @dexter-mh-lee Thanks for your review. I have updated my PR. Please help to squash the commits when applicable.

**Comment by dexter-mh-lee** [comment]
@icy Can you run this script locally https://github.com/linkedin/datahub/blob/master/docker/quickstart/generate_docker_quickstart.sh This makes sure the quickstart docker compose files picks up your changes. This will resolve the build failure.

**Comment by icy** [comment]
> @icy Can you run this script locally https://github.com/linkedin/datahub/blob/master/docker/quickstart/generate_docker_quickstart.sh This makes sure the quickstart docker compose files picks up your changes. This will resolve the build failure.
Thanks a lot. I didn't know that. I will do and keep you posted.

**Comment by icy** [comment]
> Can you try merging with latest master to resolve the build issue?

My bad... I did a rebase, and now the workflow is in waiting status. Could you please help to approve new Github-acton workflows on my branch? 

Thanks a lot.

**Comment by icy** [comment]
There was a problem with my local configuration, and the `Gitea` was shown in the last commit. This is quite annoying, let me fix that :(

**Comment by icy** [comment]
I have created new PR here https://github.com/linkedin/datahub/pull/4031 . Sorry for my noise.

**Pull request closed without merging**